webpack/scenes/Content/ContentConfig.js

```diff
@@ -142,6 +144,7 @@ export default [
       singularLowercase: __('Ansible collection'),
       pluralLabel: 'ansible_collections',
       singularLabel: 'ansible_collection',
+      capsuleCountLabel: 'ansible.collection_version',
```

Comment:
Can this be fixed on the Ruby side so that it gets translated from Pulp words to Katello words?

I thought it was `ansible.collection`... `PULPCORE_CONTENT_TYPE = "ansible.collection".freeze`.

We can change that to `ansible.collection_version`.

Reply:
Updated 👍🏼